core/src/components/checkbox/checkbox.scss

```diff
@@ -287,6 +298,46 @@ input {
   @include transform-origin(center, top);
 }
 
+// Justify Content
```

Comment:
I didn't make changes to any of this code I just moved it lower so it would take priority over the `justify-content` I specified in `label-placement-end`.